proto/rollkit/da.proto

```
@@ -0,0 +1,148 @@
+syntax = "proto3";
+package da;
```

Buf (1.47.2)

Files with package "da" must be within a directory "da" relative to root but were in directory "rollkit".

(PACKAGE_DIRECTORY_MATCH)